Components/ListSearchResults/_style.scss

```diff
@@ -1,13 +1,51 @@
 [is='flynt-list-search-results'] {
-  margin: $component-spacing-desktop 0;
+  margin: $component-spacing-mobile 0;
 
-  .container {
-    @include center;
+  @media (min-width: $breakpoint-tablet) {
+    margin: $component-spacing-tablet 0;
+  }
 
-    max-width: $content-max-width;
+  @media (min-width: $breakpoint-desktop) {
+    margin: $component-spacing-desktop 0;
+  }
 
+  .container {
     @import 'Partials/Header/style';
     @import 'Partials/Pagination/style';
     @import 'Partials/Results/style';
+
+    max-width: $content-max-width + $container-padding-mobile *2;
+
+    @media (min-width: $breakpoint-tablet) {
+      max-width: $content-max-width + $container-padding-tablet *2;
+    }
+
+    @media (min-width: $breakpoint-desktop) {
+      max-width: $content-max-width + $container-padding-desktop *2;
```

Comment:
I think it might be easier to just have another wrapping container and give it the class `centerContentMaxWidth` in HTML, so you will be able to remove all this css.